packages/cli/src/commands/doctor/healthchecks/androidSDK.js

```diff
@@ -2,15 +2,58 @@ import chalk from 'chalk';
 import {logManualInstallation} from './common';
 import versionRanges from '../versionRanges';
 import {doesSoftwareNeedToBeFixed} from '../checkInstallation';
+import {exec} from 'child_process';
+
+const installMessage = `Read more about how to update Android SDK at ${chalk.dim(
+  'https://developer.android.com/studio',
+)}`;
 
 export default {
   label: 'Android SDK',
-  getDiagnosticsAsync: async ({SDKs}) => ({
-    needsToBeFixed: doesSoftwareNeedToBeFixed({
-      version: SDKs['Android SDK']['Build Tools'][0],
-      versionRange: versionRanges.ANDROID_NDK,
-    }),
-  }),
+  getDiagnosticsAsync: async ({SDKs}) => {
+    let sdks = SDKs['Android SDK'];
+
+    // This is a workaround for envinfo's Android SDK check not working on
+    // Windows. This can be removed when envinfo fixes it.
+    // See the PR: https://github.com/tabrindle/envinfo/pull/119
+    if (sdks === 'Not Found' && process.platform !== 'darwin') {
+      sdks = await new Promise(resolve => {
+        exec(
+          process.env.ANDROID_HOME
+            ? `${process.env.ANDROID_HOME}/tools/bin/sdkmanager --list`
+            : 'sdkmanager --list',
+          (err, stdout) => {
+            if (err) {
+              resolve('Not Found');
```

Comment:
Should this be `reject()`?

Reply:
I was wondering same thing. This probably should be `reject`.